bindata/network/ovn-kubernetes/common/004-rbac-control-plane.yaml

```diff
@@ -144,6 +145,10 @@ rules:
   - network-attachment-definitions
   - multi-networkpolicies
   verbs: ["list", "get", "watch"]
+- apiGroups: [ "k8s.cni.cncf.io" ]
```

Comment:
move to under feature gate section in L159

Reply:
also double check this is the only place we need to do this for HCP ad OCP.. I can never remember this point

Reply:
> move to under feature gate section in L159

Done
UDN CRD RBAC moved as well

> also double check this is the only place we need to do this for HCP ad OCP.. I can never remember this point

Could you please refer me to some source so I could verify it?

Reply:
From what I understand, the UDN RBAC and CRD will take place for both configurations because the files are under common directory.